spec/unit/analytics/ga4_import/report_request_builder_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+
+RSpec.describe Analytics::Ga4Import::ReportRequestBuilder do
+  describe "#report_request" do
+    it "builds the correct RunReportRequest object" do
+      analytics_data = described_class.new(0, 100).report_request
+      start_date = Date.today.prev_day - 13
+      end_date = Date.today.prev_day
+
+      expect(analytics_data).to have_attributes(
+        property: "properties/330577055",
+        offset: 0,
+        limit: 100,
+        metric_aggregations: [],
+        order_bys: [],
+        currency_code: "",
+        keep_empty_rows: false,
+        return_property_quota: true,
```

Comment:
These attributes don't reference anything in our code so shouldn't be asserting against it.